nominees/portableutm.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "PortableUTM",
+  "aliases": [
+    ""
+  ],
+  "description": "PortableUTM is an Open Source Initiative for enabling safe and regulations compliant drone operations around the world. The project is publicly released under GPU/GPL v3.0 licensing for the core features and MIT permissive license for the frontend and mobile apps.",
+  "website": "https://www.portableutm.com",
+  "license": [
+    {
+      "spdx": "AGPL-3.0",
+      "licenseURL": "https://github.com/portableutm/dronfiesuss/blob/master/LICENSE"
+    },
+    {
+      "spdx": "MIT",
+      "licenseURL": "https://github.com/portableutm/webapp/commit/e08986b2567063ace1643b4673408bfff0e07cb4"
+    }
```

Comment:
@nathanbaleeta @lacabra - This second license refers to a separate repository. In the past, we have had separate entries in the DPGA registry for each repository (ie. one for DronefieUSS and one for the webapp). I would advocate for removing this second license entry here and adding a new entry for the webapp if applicable.

Reply:
Resolved in commit c30de06